charts/ip-blocker/values.yaml

```diff
@@ -76,6 +85,52 @@ elasticSearch:
             }
           }
         }
+      queryElasticCityOrg.json: |
```

Comment:
I'd extract this from here to just provide in the repo safe defaults or something useful but simpler.